tests/tasks/generators/clusterloader/load.yaml

```diff
@@ -105,7 +105,7 @@ spec:
       cat $(workspaces.source.path)/perf-tests/clusterloader2/pkg/prometheus/manifests/prometheus-prometheus.yaml
       fi
   - name: run-loadtest
-    image: public.ecr.aws/kit/clusterloader2:0213bea
+    image: 197575167141.dkr.ecr.us-west-2.amazonaws.com/clusterloader2:latest
```

Comment:
why is this changed ? is this a version upgrade ?

Reply:
I have applied an internal patch to fix the failures we are seeing for EKS scale tests. I have a PR in upstream and in the process to make it to upstream.

Reply:
we have been using this patch for a while now.

Reply:
okay. you can link that PR here for tracking.

Reply:
https://github.com/kubernetes/perf-tests/pull/2178